keda/src/components/scaledobjects/List.tsx

```diff
@@ -0,0 +1,102 @@
+import { Link, ResourceListView } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { useKedaInstalled } from '../../hooks/useKedaInstalled';
+import { ScaledObject } from '../../resources/scaledobject';
+import { TriggerAuthentication } from '../../resources/triggerAuthentication';
+import { NotInstalledBanner } from '../common/CommonComponents';
+
+export function ScaledObjectsList() {
```

Comment:
Since the number of columns is more the age and authentications column is not rendering properly, horizontal scroll is also not supported. We can group Min,Max to a single column and similarly for scaleTargetKind and scaleTargetName. WDYT?

<img width="1508" alt="image" src="https://github.com/user-attachments/assets/a441656c-44db-44b9-b72c-923e59546cc9" />